core/src/main/scala/org/broadinstitute/dsde/rawls/user/UserService.scala

```diff
@@ -308,6 +312,53 @@ class UserService(protected val userInfo: UserInfo, val dataSource: SlickDataSou
     gcsDAO.deleteProject(projectName).map(_ => RequestComplete(StatusCodes.OK))
   }
 
+  def registerBillingProject(projectName: RawlsBillingProjectName, owner: RawlsUserEmail): Future[PerRequestMessage] = {
+    val cromwellAuthBucketName = s"cromwell-auth-${projectName.value}"
```

Comment:
this will be returned by gpalloc so it can come in here as a parameter rather than constructing it with string magic. see https://github.com/broadinstitute/gpalloc/pull/15